README.md

```diff
@@ -67,6 +67,31 @@ distinguished by their unique VIN numbers:
       ]
     }
 
+## Homekit
```

Comment:
I think this should say "HomeLink"? Some other instances of this below…

Reply:
Homekit on the brain! Fixed these.